### PR TITLE
Events section style update

### LINKED ISSUE #63: Events Section Styling Enhacements
### Describe the Current Behavior/Feature:

 - Events cards full-width center on mobile
 - Center text on mobile
 - Adjust google calendar to scale better (make it fluid but keep aspect ratio)

Currently.. 
Screenshot 2024-01-30 at 7 43 40 PM

https://github.com/codeforpdx/codepdx_website/assets/114430929/46a9448b-9a7c-47cb-b39b-973770101ddd

---

### Proposed Implementation:

- Center boxes in mobile view only and allow text to take up more screen space. 
- Potentially use % units to scale the calendar and remove the breakpoints.

### PR DESCRIPTION
**Pull request by fighawkins**
## This PR:

Resolves #63 

---

### CODE REVIEW

File: src/components/home/Events.jsx [hunk 1]
```diff
@@ -6,15 +6,44 @@ import Link from '@mui/material/Link';
 import Paper from '@mui/material/Paper';
 import Typography from '@mui/material/Typography';
 
+const mobileStyle = {
```
Comment (Jared-Krajewski) [inline src/components/home/Events.jsx]: I don't think either of these are doing anything so they can be removed.

**Review by Jared-Krajewski — CHANGES_REQUESTED** [review]
One comment on code that does nothing otherwise it looks good!

File: src/components/home/Events.jsx (hunk 1 above)
Comment (fighawkins) [reply src/components/home/Events.jsx]: are you referring to lines 6 and 7? I may be wrong but I believe they are necessary. I've tried removing them and the page crashes. We are using the Paper and Typography components from mui, so its just a simple import.  Let me know if I'm misunderstanding.

Comment (Jared-Krajewski) [reply src/components/home/Events.jsx]: sorry for confusion, the tag for this starts on line 9. was referring to `mobileStyle` and `desktopStyle`. The imports are necessary.

Comment (fighawkins) [reply src/components/home/Events.jsx]: oh gotcha, okay, i've removed the unneeded lines, thank you.

**Review by Jared-Krajewski — APPROVED** [review]
🚀

**Pull request merged**